app/src/main/java/com/example/neighborly/NeighborsRequestListAdapter.java

```diff
@@ -0,0 +1,77 @@
+package com.example.neighborly;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NeighborsRequestListAdapter extends BaseAdapter {
```

Comment:
What do you think about combining all the list adapters? Maybe creating a list adapter that all the others will extend? 
To avoid code duplication..

Reply:
Thought about it, however I've encountered with a few issues on that part:
1. The design of the Request_details is different than the Neighbor_details by the design of Gal's/Yali, Now it looks the same, but when I'll have the rounded rectangle frame it will look different.
2. The click listeners is different, one opens the private chat and one opens the request.
3. We want to add the "Badges" to the neighbors in a later run, and it will be shown in the neighbor_details.